src/services/cdn.ts

```diff
@@ -139,6 +139,13 @@ export class K8SCDNConnector implements ICDNConnector {
     urls = urls.map((item) => {
       return `/${item}`;
     });
+
+    // perform wildcard invalidations when invalidating >= 250 objects
+    if (urls.length >= 250) {
+      const wildcardPatternURL = '/' + pathPrefix + '/*';
```

Comment:
Should we have an assertion here that pathPrefix isn't falsey? Are there circumstances where it would be okay for pathPrefix to be null in the first place?